Document optional named vectors (#106)
qdrant/docs: qdrant/v1.2.x/points.md

*Available since v0.10.0*

> If the collection was created with multiple vectors, each vector data can be provided using the vectors name:

Reviewer: vector's

## Modify points

You can modify a point in two ways. You can either modify the vectors or modify
the payload. For both are several methods.

### Update vectors

*Available since v1.2.0*

This method updates the specified vectors on the given points. Unspecified
vectors are kept unchanged. All given points must exist.

REST API ([Schema](https://qdrant.github.io/qdrant/redoc/index.html#operation/update_vectors)):

```http
PUT /collections/{collection_name}/points/vectors

{
    "points": [
        {
            "id": 1,
            "vector": {
                "image": [0.1, 0.2, 0.3, 0.4]
            }
        },
        {
            "id": 2,
            "vector": {
                "text": [0.9, 0.8, 0.7, 0.6, 0.5, 0.4, 0.3, 0.2]
            }
        }
    ]
}
```

```python
client.update_vectors(
    collection_name="{collection_name}",
    points=[
        models.PointStruct(
            id=1,
            vector={
                "image": [0.1, 0.2, 0.3, 0.4],
            },
        ),
        models.PointStruct(
            id=2,
            vector={
                "text": [0.9, 0.8, 0.7, 0.6, 0.5, 0.4, 0.3, 0.2],
            },
        ),
    ]
)

> ```

Author: Python client must be updated before this works.

> To update points without keeping existing vectors intact, see [uploading

Reviewer 2: ```suggestion
To update points and replace all of its vectors, see [uploading
```

Author: I wrote it like this because I thought 'and replace all of its vectors' wouldn't make clear that it also nullifies unspecified vectors.

But this is definitely easier to understand, thanks for the suggestion.

points](#upload-points).

### Delete vectors

*Available since v1.2.0*

This method deletes just the specified vectors from the given points. Other
vectors are kept unchanged. Points are never deleted.

REST API ([Schema](https://qdrant.github.io/qdrant/redoc/index.html#operation/deleted_vectors)):

```http
POST /collections/{collection_name}/points/vectors/delete

{

>     "points": [0, 1],

Reviewer 2: Should be consistent between http and python examples
```suggestion
    "points": [0, 3, 100],
```

    "vectors": ["text", "image"]
}
```

```python
client.delete_vectors(
    collection_name="{collection_name}",
    points_selector=models.PointIdsList(
        points=[0, 3, 100],
    ),
    vectors=["text", "image"]
)

> ```

Author: Python client must be updated before this works.

## Modify points

You can modify a point in two ways. You can either modify the vectors or modify

> the payload. For both are several methods.

Reviewer: "There are two ways to do this." ......or....."There are several ways to do this."

Author: This tries to say:
- there are two ways to modify a point (modify its vectors, or modify its payload)
- there are several methods to modify these vectors and payloads

What would be a good replacement sentence? What about this?

> To change a point, you can modify its vectors or its payload. There are several ways to do this.

Reviewer 3: I like your last suggestion.

> To change a point, you can modify its vectors or its payload. There are several ways to do this.

My brain had trouble parsing "For both are several methods".

The second is to modify the payload, for which there are several methods.

### Update vectors

> *Available since v1.2.0*

Reviewer: Available as of v1.2.0. Since is used in other cases.

Author: Currently the documentation only uses 'since' versus 'as of'.

Reviewer 3: I agree, I'd prefer a PR changing all occurrences in one go :+1:
